.github/workflows/pr.yaml

```diff
@@ -77,6 +78,10 @@ jobs:
           path: /tmp/deadlock-detector-*
           if-no-files-found: ignore
 
+  # testLatestDep is intentionally not included in the PR workflow
+  # because it can break randomly when a new library version is released
+  # which can be confusing for contributors
```

Comment:
I'm wondering how this is different from running muzzle in the PR workflow..

Reply:
Muzzle only runs for the instrumentation code; latestDepTest often detects problems in the test code (or slight changes in emitted telemetry)

Reply:
👍 updated comment to reflect